### ISSUE TITLE
Asynchronous Distributed Bayesian Optimization at HPC Scale

### ISSUE THREAD

**Issue by nabenabe0928** [nabenabe0928/reading-list#115]
[Asynchronous Distributed Bayesian Optimization at HPC Scale](https://arxiv.org/abs/2207.00479)

Asynchronous BO that scales up to 4096 workers

## Main points
Basically, the authors tried to address the computational time and local optima issues.

- each worker executes BO with different control parameters to achieve a good trade-off of control parameters
- each worker shares the observations but uses only a certain number of observations

The undersampling is done by:
1. Split observations into six parts based on the quantile of objective values,
2. sample $N / 5$ observations from each part,
3. concatenate and use them as a dataset

Then each sample will be re-sampled anyways at some points because each sample inside each part is a uniform sampling.

- use random forest surrogate with a special treatment for the stabilization of variance measure as in Frank's SMAC
- Use only $\log_2 D$ dimensions among $D$ dimensions for training of a surrogate 

## Experiments

Basically, only performance over time. Otherwise, runtime analysis.
They just showed that their method CAN run with many workers.
